content/en/continuous_integration/static_analysis/_index.md

```diff
@@ -97,29 +96,54 @@ Configure the following environment variables:
 
 Provide the following inputs:
 
-| Name       | Description                                                                                                                | Required | Default         |
-|------------|----------------------------------------------------------------------------------------------------------------------------|----------|-----------------|
-| `service`  | The name of the service to tag the results with.                                                                           | Yes      |                 |
-| `env`      | The environment to tag the results with. `ci` is a helpful value for this input.                                           | No       | `none`          |
+| Name        | Description                                                                                                                | Required | Default         |
+|-------------|----------------------------------------------------------------------------------------------------------------------------|----------|-----------------|
+| `service`   | The name of the service to tag the results with.                                                                           | Yes      |                 |
+| `env`       | The environment to tag the results with. `ci` is a helpful value for this input.                                           | No       | `none`          |
+| `cpu_count` | Set the number of CPUs used by the analyzer. Defaults to the number of CPUs available.                                     | No       |                 |
+
+<div class="alert alert-info">
+  Add a `--performance-statistics` flag to your static analysis command to get execution time statistics for analyzed files.
+</div>
+
+Select an analyzer for your architecture and OS:
+
+| Architecture | OS        | Name                                                    |
+|--------------|-----------|---------------------------------------------------------|
+| `aarch64`    | `Darwin`  | `datadog-static-analyzer-aarch64-apple-darwin.zip`      |
+| `aarch64`    | `Linux`   | `datadog-static-analyzer-aarch64-unknown-linux-gnu.zip` |
+| `x86_64`     | `Darwin`  | `datadog-static-analyzer-x86_64-apple-darwin.zip`       |
+| `x86_64`     | `Linux`   | `datadog-static-analyzer-x86_64-pc-windows-msvc.zip`    |
+| `x86_64`     | `Windows` | `datadog-static-analyzer-x86_64-unknown-linux-gnu.zip`  |
```

Comment:
@dastrong let's also have a link to the latest

Reply:
The user should not have to guess the link themselves. We should give the links to them.

Just have a fourth column with a link to the download.

Example of link: `https://github.com/DataDog/datadog-static-analyzer/releases/latest/download/datadog-static-analyzer-x86_64-unknown-linux-gnu.zip` - just replace `datadog-static-analyzer-x86_64-unknown-linux-gnu.zip`. It gives a link to the latest release if people want to integrate it in their CI.